app/models/lettings_log.rb

```diff
@@ -706,6 +706,20 @@ def process_postcode_changes!
     self.la = inferred_la if inferred_la.present?
   end
 
+  def scheme_has_multiple_locations?
```

Comment:
There's more to do to properly fix the merge conflict here - I edited this function in the other file, while you've moved it here, and I think you've now got a different version of it in each file?

Git has just shown the conflict it knows about - i.e. lines where you removed something and main had changed it - it doesn't know that you'd moved the function to here.

With that kind of conflict, you need to look at the actual changes and ensure you've built them in to your work - so remove the version in the other file (because you've moved the function) but replicate the changes that had been made to it here so they aren't lost.

You may need similar changes to the `scheme_has_large_number_of_locations?` function below as well.

Reply:
Amended